cms/auth/checks.py

```diff
@@ -0,0 +1,95 @@
+from collections.abc import Iterable
+from typing import TYPE_CHECKING, Any
+
+from django.conf import settings
+from django.core.checks import Error, register
+
+if TYPE_CHECKING:
+    from django.apps import AppConfig
+    from django.conf import LazySettings
+
+
+def _check_aws_cognito(errors: list, settings_obj: "LazySettings") -> None:
+    aws_cognito_enabled = getattr(settings_obj, "AWS_COGNITO_LOGIN_ENABLED", False)
+    if aws_cognito_enabled:
+        cognito_settings: list[tuple[str, str, str]] = [
+            ("AWS_COGNITO_USER_POOL_ID", "eu-west-2_xxxxxxxxx", "auth.E001"),
+            ("AWS_COGNITO_APP_CLIENT_ID", "1234567890abcdefghijklmnop", "auth.E002"),
```

Comment:
Naming is hard, and this hint could do with a more realistic example or a suggestive value (like `the-cognito-app-client-id`?)